lib/miq_automation_engine/models/miq_ae_yaml_import.rb

```diff
@@ -8,6 +8,10 @@ def initialize(domain, options)
     @domain_name = domain
     @options     = options
     @restore     = @options.fetch('restore', false)
+    tenant_id    = @options.fetch('tenant_id', Tenant.root_tenant.try(:id))
```

Comment:
Ok. Then fetch on tenant, and if that fails, do a fetch on id. But don't lookup the root by id. That is silly